scripts/cover.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+# Run the test suite so that:
+# --> -x: Stops on first error or failure
+# --> -s: Outputs all diagnostic information
+# --> Other parameters: Track coverage and output report as XML file
```

Comment:
Many thanks for this added documentation! :smile:

Note: `-x` is not used in this command, so perhaps that part of the documentation should be removed.

Reply:
Yes, great point, thanks for catching this! I initially added the `-x` and then realized that it should not be used in this context. I deleted that comment and then added another comment to explain that we are tracking both branch and statement coverage of the test suites.